globus_sdk/transfer/client.py

```diff
@@ -1604,12 +1604,95 @@ def endpoint_manager_task_list(self, num_results=10, **params):
                 an iterable of :class:`GlobusResponse
                 <globus_sdk.response.GlobusResponse>`
 
+        **Filters**
+
+        The following filters are supported (passed as keyword arguments in ``params``).
+        For any query that doesn’t specify a filter_status that is a subset of
+        ("ACTIVE", "INACTIVE"), at least one of filter_task_id or filter_endpoint is
+        required.
+
+        ====================== ================ ========================
+        Query Parameter        Filter Type      Description
+        ====================== ================ ========================
+        filter_status          equality list    |filter_status|
+        filter_task_id         equality list    |filter_task_id|
+        filter_owner_id        equality         |filter_owner_id|
+        filter_endpoint        equality         |filter_endpoint|
+        filter_is_paused       boolean equality |filter_is_paused|
+        filter_completion_time datetime range   |filter_completion_time|
+        filter_min_faults      int              |filter_min_faults|
+        filter_local_user      equality         |filter_local_user|
+        ====================== ================ ========================
+
+        .. |filter_status| replace::
+           Comma separated list of task statuses.  Return only tasks with any of the
+           specified statuses. Note that in-progress tasks will have status "ACTIVE" or
+           "INACTIVE", and completed tasks will have status "SUCCEEDED" or "FAILED".
```

Comment:
Under scrutiny it looks funny, but I see the Sphinx docs use this sort of weird style with the three-space indent, to match the `|x|` text.

Reply:
That's fine. The alignment looks nice, IMO, but if you chose to do 4 spaces instead, I wouldn't bat an eye.

The table above, where you used these, is really nice to read. I'm going to double-check the text one last time before merging though, since there's a lot of doc here.